app/js/components/manageApps/ManageApps.jsx

```diff
@@ -415,6 +417,30 @@ export default class ManageApps extends React.Component {
     };
   }
 
+  checkForUpdates() {
+    const installedAddons = this.state.appList;
+    const updatesAvailable = [];
+    axios.get(`https://addons.openmrs.org/api/v1//addon`).then((response) => {
+      this.state.appList.forEach((addon) => {
+        response.data.forEach((result) => {
+          if (addon.appDetails.name == result.name) {
+            result.latestVersion > addon.appDetails.version ?
+              updatesAvailable.push(addon.appDetails.name)
+              :
+              null
+            return;
+          }
+        });
+      });
+
+      this.setState({ updatesAvailable });
+    }).catch(
+      (error) => {
```

Comment:
`error` returned here will contain the response, status, header and some other stuffs. I'm not sure if passing `error` into `toastr.error()` will deconstruct it to get the error message. If not, you might want to deconstruct the `error` and pass in only the data to toastr.

Something like
`.catch(({ response }) => {
  toastr.error(response.data.theErrorMessageKey)
})`

You can consle.log the error to see what it contains.

Reply:
`toastr.error()` still deconstructs to get the error message

Reply:
Oh great! Nice job 👍